tests/test_helpers.py

```diff
@@ -17,6 +17,19 @@
       "--date", "2020-04-29", "-c", "afriend", "-d", "more work",
       "-m", "ascopatz", "-g", "SymPy-1.1"],
      "20ly_newprojectum has been added in projecta\n"),
+    (["helper", "a_proposal", "an empty new proposal","100.0","To understand numbers"],
+    "20_an_empty_new_proposal has been added in proposals\n"),
+    (["helper", "a_proposal", "a sparse new proposal","500.0","To destroy numbers",
+      "-a", "-c", "-p", "--cppflag", "--other_agencies", "-i",
+      "--months_academic", "--months_summer", "-s", "-f", "-n", "this is a minimal proposal"],
+     "20_a_sparse_new_proposal has been added in proposals\n"),
+    (["helper", "a_proposal", "a dense new proposal","1000.0","To create numbers",
+      "--begin_date", "2020-09-22", "--duration", "3", "--due_date", "2020-12-29",
+      "-a", "Kurt Godel", "-c", "Bitcoin" "-p", "sbillinge", "--cppflag", "False",
+      "--other_agencies", "Flatland", "-i", "Columbia", "--months_academic", "6",
```

Comment:
since the tests act a bit like examples, could you give the institution as columbiau (institution-id) which is what I want to put in there the most.  Also, is end_date missing?

Reply:
will do for the institution. the `end_date` is internally computed using 

`begin_date = date_parser.parse(rc.begin_date).date()
  pdoc.update({'end_date': begin_date + relativedelta(months = rc.duration)})`

Reply:
yes you are so right....my apologies!

Reply:
no problem!